pkg/apis/pravega/v1alpha1/zz_generated.deepcopy_test.go

```diff
@@ -0,0 +1,316 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package v1alpha1_test
+
+import (
+	"github.com/pravega/pravega-operator/pkg/apis/pravega/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("PravegaCluster DeepCopy", func() {
+	Context("with defaults", func() {
+		var str1, str2 string
+		var str3, str4 v1.PullPolicy
+
+		var p1, p2 *v1alpha1.PravegaCluster
```

Comment:
Could we add all variables inside a `var` block
```
var (
   str1, str2 string
   ...
)
```

Reply:
done